tests/testJointcal.py

```diff
@@ -0,0 +1,239 @@
+    for cat in catalogs:
+        good = (cat.src.get('base_PsfFlux_flux')/cat.src.get('base_PsfFlux_fluxSigma')) > fluxlimit
+        # things the classifier called stars.
+        good &= (cat.src.get('base_ClassificationExtendedness_value') == 0)
+        matches = table.matchRaDec(reference, cat.src[good], radius)
+        for m in matches:
+            deltas[m[0].getId()].append(m[2])
+    return deltas
+
+
+# NOTE: Stolen from meas_astrom.fitTanSipWcs.
+def updateSourceCoords(wcs, catalog):
+    """Update coords in a collection of sources, given a WCS."""
+    schema = catalog.src.schema
+    srcCoordKey = table.CoordKey(schema["coord"])
+    for src in catalog.src:
+        src.set(srcCoordKey, wcs.pixelToSky(src.getCentroid()))
+
+
+def compute_rms(reference, catalogs, fluxlimit=100):
+    """Match all catalogs to compute the RMS, for all detections above some flux limit."""
+
+    old_relative = make_match_dict(catalogs[0].src, catalogs[1:])
+    old_absolute = make_match_dict(reference, catalogs, fluxlimit=fluxlimit)
+    # Update coordinates with the new distortion corrections.
+    for cat in catalogs:
+        updateSourceCoords(cat.wcs.getWcs(), cat)
+    new_relative = make_match_dict(catalogs[0].src, catalogs[1:])
+    new_absolute = make_match_dict(reference, catalogs)
+
+    if doPlot:
+        old_rms_relative = rms_per_star(old_relative)
+        old_rms_absolute = rms_per_star(old_absolute)
+        new_rms_relative = rms_per_star(new_relative)
+        new_rms_absolute = rms_per_star(new_absolute)
+        print(len(catalogs))
+        print("relative RMS:", rms_total(old_relative)*206265, rms_total(new_relative)*206265)
+        print("absolute RMS:", rms_total(old_absolute)*206265, rms_total(new_absolute)*206265)
+        plot_deltas(old_rms_relative, old_rms_absolute, new_rms_relative, new_rms_absolute)
+
+    return rms_total(new_relative), rms_total(new_absolute)
+
+
+def plot_deltas(old_rms_relative, old_rms_absolute, new_rms_relative, new_rms_absolute):
+    import matplotlib.pyplot as plt
+    plt.ion()
+
+    plt.figure()
+
+    color_rel = 'green'
+    color_abs = 'blue'
+    plt.title('relative vs. absolute: %d vs. %d'%(len(old_rms_relative), len(old_rms_absolute)))
+
+    # 206265 arcseconds per radian
+    plt.hist(old_rms_relative*206265, color=color_rel, ls='dashed', label='old rel',
+             histtype='step', bins=30, lw=1.5)
+    plt.hist(new_rms_relative*206265, color=color_rel, ls='solid', label='new rel',
+             histtype='step', bins=30, lw=1.5)
+
+    plt.hist(old_rms_absolute*206265, color=color_abs, ls='dashed', label='old abs',
+             histtype='step', bins=30, lw=1.5)
+    plt.hist(new_rms_absolute*206265, color=color_abs, ls='solid', label='new abs',
+             histtype='step', bins=30, lw=1.5)
+
+    plt.xlabel('arcseconds')
+    plt.legend(loc='best')
+
+    # So one can muck-about with things after plotting...
+    import ipdb
+    ipdb.set_trace()
+
+
+def suite():
```

Comment:
Judging from SQR-012, the correct thing to do replace that whole block with this, right?
```
if __name__ == "__main__":
    lsst.utils.tests.init()
    unittest.main()
```